configs/templates/generated_interfaces_for_adapters/log/LogTemplate.gen.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package istio.mixer.adapter.log;
+
+import "mixer/v1/config/descriptor/value_type.proto";
+import "mixer/configs/mixer/TemplateExtensions.proto";
+
+option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_REPORT;
+option (istio.mixer.v1.config.template.template_name) = "Log";
+
+/*
+Data from Template proto
+message Constructor {
```

Comment:
This is in the comment block, not part of this proto. I just wanted to show what we got from the original Template.proto.. removing it all together. This comment can confuse others too.